### PR TITLE
Fix state transition tooltip not showing label / path value

### PR DESCRIPTION
**Pull request by achim-k**
**User-Facing Changes**
Fix state transition tooltip not showing label / path value

**Description**
The tooltip was showing the stringified series index instead of the appropriate label / path value.

| Before                                                                                           | After (this PR)                                                                                  |
|--------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
| image | image |

### CODE REVIEW

**Comment by jtbandes** [comment]
Any idea why the color is black? Is it because of the `pointBackgroundColor: "rgba(0, 0, 0, 0.4)",`? I'm not sure if that is a regression but it seems strange.

**Review by jtbandes — APPROVED** [review]

File: packages/studio-base/src/panels/StateTransitions/messagesToDatasets.ts [hunk 1]
```diff
@@ -45,12 +45,12 @@ type Args = {
...
  */
 export default function messagesToDatasets(args: Args): ChartDatasets {
-  const { path, pathIndex, startTime, y, blocks } = args;
+  const { path, startTime, y, blocks } = args;
 
   const dataset: ChartDataset = {
     borderWidth: 10,
     data: [],
-    label: pathIndex.toString(),
+    label: path.label ?? "" ? path.label : path.value,
```
Comment (snosenzo) [inline packages/studio-base/src/panels/StateTransitions/messagesToDatasets.ts:53]: I think this should be
```suggestion
    label: path.label ? path.label : path.value,
```

**Review by snosenzo — APPROVED** [review]
one small change, otherwise looks good

File: packages/studio-base/src/panels/StateTransitions/messagesToDatasets.ts (hunk 1 above)
Comment (jtbandes) [reply packages/studio-base/src/panels/StateTransitions/messagesToDatasets.ts:53]: I suspect eslint forced the issue here... but yeah this looked weird to me too.

**Pull request merged**